packages/venia-ui/lib/components/CartPage/ProductListing/quantity.css

```diff
@@ -66,6 +61,15 @@
     composes: root from '../../Icon/icon.css';
 }
 
+.button:hover .icon svg {
+    stroke: rgb(var(--venia-global-color-gray-900));
+}
+
+.button:disabled .icon svg,
+.button:disabled:hover .icon svg {
```

Comment:
Ah, were we not able to pass a `--stroke` here for some reason?

Reply:
Yeah, it's hard to pass in the custom variable for `--stroke` here, possibly due to the `.icon` override from above, which also declares a different value for `--stroke`

Reply:
That icon override was overriding the `root` class with the `root` class from icon 🤯   Seemed redundant, so I deleted the override and was able to get variable cascade to work as expected.

```css
.button:hover {
    --stroke: rgb(var(--venia-global-color-gray-900));
}

.button:disabled,
.button:disabled:hover {
    --stroke: rgb(var(--venia-global-color-gray-400));
}
```